README.md

```diff
@@ -25,7 +25,7 @@ If you are using Windows and python was installed using the [python installer](h
 
 From within the python environment described in the [Environment](#environment) section, run the command:
 
```

Comment:
Maybe mention that on windows, use python -m pip install ...

Reply:
Added a note about pip usage on windows.